ext/json/json.c

```diff
@@ -116,6 +120,7 @@ static PHP_MINIT_FUNCTION(json)
 	/* common options for json_decode and json_encode */
 	PHP_JSON_REGISTER_CONSTANT("JSON_INVALID_UTF8_IGNORE", PHP_JSON_INVALID_UTF8_IGNORE);
 	PHP_JSON_REGISTER_CONSTANT("JSON_INVALID_UTF8_SUBSTITUTE", PHP_JSON_INVALID_UTF8_SUBSTITUTE);
+	PHP_JSON_REGISTER_CONSTANT("JSON_THROW_EXCEPTIONS", PHP_JSON_THROW_EXCEPTIONS);
```

Comment:
With JSON_ERRMODE_ as a common prefix, it is easy to understand the purpose for the constants and you never will not have any naming conflicts in the future. Anyone with an IDE without an autocomplete can create "aliases" for that in their own projects like:

    define('JDO', JSON_ERRMODE_EXCEPTION | JSON_BIGINT_AS_STRING);
    $data = json_decode($json, JDO);

In my opinion clear naming (with reasonable length) is more important than short code.